README.md

```diff
@@ -1 +1,118 @@
 # OKRs 2018 Q4
+
+## Bring 'Code as Data' to market with enterprises
+
+### Ensure gitbase interoperability with 3rd party tools [DR]
+
+- [ ] Drivers for in-house languages (Python, Go, Java).
+- [ ] mysqldump
+- [ ] grafana
+- [ ] Tableau
+- [ ] Drivers for other popular languages (JavaScript, Ruby, .NET, PHP).
+
+### Get gitbase enterprise ready [DR]
+
+- [ ] Authentication and authorization abstractions prepared for future integration with LDAP and Kerberos
+- [ ] Basic MySQL auth implementation
+- [ ] Audit logs (connections, queries)
+
+### Get web interfaces enterprise ready [APPS]
+
+- [ ] gitbase-web SSO (protocol TBD)
+- [ ] bblfsh-web SSO (protocol TBD)
+
+### Gitbase research [DR]
+
+- [ ] Support updates in backing data (indexes, etc)
+- [ ] Speed getting UAST, possibly with the usage of persistent caches for UAST and others
```

Comment:
Reword to `Improve queries that are using UAST functions`

Reply:
Rephrased to `Speed up retrieving UASTs`